validation/policies/io/konveyor/forklift/openstack/cpu_shares.rego

```diff
@@ -0,0 +1,14 @@
+package io.konveyor.forklift.openstack
+
+has_cpushares_enabled {
+	input.flavor.extraSpecs["quota:cpu_shares"]
+}
+
+concerns[flag] {
+	has_cpushares_enabled
```

Comment:
shouldn't it be a boolean value when we get here? I also see this in host_devices.rego below so maybe there's something about this syntax that I don't realize

Reply:
Ok, I upgraded the opa version and used your latest suggestion.